scss/_forms.scss

```diff
@@ -190,13 +190,13 @@ textarea.form-control {
 .form-row {
   display: flex;
   flex-wrap: wrap;
-  margin-right: -5px;
-  margin-left: -5px;
+  margin-right: -($form-grid-gutter-width / 2);
+  margin-left: -($form-grid-gutter-width / 2);
 
   > .col,
   > [class*="col-"] {
-    padding-right: 5px;
-    padding-left: 5px;
+    padding-right: ($form-grid-gutter-width / 2);
```

Comment:
I don't think the brackets are needed here, can you check if you can remove them?

Reply:
I don't think so too, but they are also present in [here](https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_grid-framework.scss#L11).

Reply:
Good point. I just added #27768 to remove those redundant brackets. You can also remove it here.